### PR TITLE
(Op) number token

### PR DESCRIPTION
**Pull request by SoluMilken**
1. int_token: replace integer with `_int_`
2. float_token: replace float with `_float_`
3. int_token_with_space: replace integer with `  _int_  `
4. float_token_with_space: replace float with `  _float_  `

### CODE REVIEW

**Comment by absolutelyNoWarranty** [comment]
這應該可以refactor成regex token吧？ 就像text normalizer那邊 https://github.com/Yoctol/text-normalizer/blob/master/text_normalizer/factory/replace_pattern_with_token.py

**Comment by SoluMilken** [comment]
我想說多寫幾個ops 後再refactor

File: uttut/pipeline/ops/float_token.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+    E.g.
+        I have 10.7 dollars. -> I have _float_ dollars.
+    """
+
+    REGEX_PATTERN = re.compile(r"(?<![\.\d])\d+\.\d+(?![\.\d])")
+    TOKEN = FLOAT_TOKEN
+
+    def _gen_forward_replacement_group(self, input_str: str):  # type: ignore
+        return super(IntToken, self)._gen_forward_replacement_group(  # type: ignore
```
Comment (absolutelyNoWarranty) [inline uttut/pipeline/ops/float_token.py]:
```suggestion
        return super()._gen_forward_replacement_group(  # type: ignore
```

File: uttut/pipeline/ops/float_token_with_space.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+    E.g.
+        I have 10.7 dollars. -> I have  _float_  dollars.
+    """
+
+    REGEX_PATTERN = re.compile(r"(?<![\.\d])\d+\.\d+(?![\.\d])")
+    TOKEN = FLOAT_TOKEN_WITH_SPACE
+
+    def _gen_forward_replacement_group(self, input_str: str):  # type: ignore
+        return super(    # type: ignore
```
Comment (absolutelyNoWarranty) [inline uttut/pipeline/ops/float_token_with_space.py]: super()

File: uttut/pipeline/ops/float_token.py (hunk 1 above)
Comment (SoluMilken) [reply uttut/pipeline/ops/float_token.py]: 不行 我要抓他爺爺的class

File: uttut/pipeline/ops/float_token_with_space.py (hunk 2 above)
Comment (SoluMilken) [reply uttut/pipeline/ops/float_token_with_space.py]: 不行 我要抓他爺爺的class

**Pull request merged**